modules/sql/README.md

```diff
@@ -0,0 +1,92 @@
+# GKE module
+
+Module for provision Cloud SQL on Google Cloud
+
+## Requirements
+
+| Name      | Version   |
+| --------- | --------- |
+| terraform | >= 0.12   |
+| google    | >= 3.21.0 |
+| random    | >= 2.2.1  |
+
+## Providers
+
+| Name   | Version   |
+| ------ | --------- |
+| google | >= 3.21.0 |
+| random | >= 2.2.1  |
+
+## Inputs
+
+| Name              | Description                                                                                                                                                                                              | Type                | Default         | Required |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | --------------- | :------: |
+| activation_policy | This specifies when the instance should be active. Can be either ALWAYS, NEVER or ON_DEMAND.                                                                                                             | `string`            | `"ALWAYS"`      |    no    |
+| auto_size         | Configuration to increase storage size automatically. Note that future terraform apply calls will attempt to resize the disk to the value specified in disk_size - if this is set, do not set disk_size. | `bool`              | `true`          |    no    |
+| availability_type | This specifies whether a PostgreSQL instance should be set up for high availability (REGIONAL) or single zone (ZONAL).                                                                                   | `string`            | `"ZONAL"`       |    no    |
```

Comment:
Only for PostgreSQL?

Reply:
The availability type of the Cloud SQL instance, high availability (REGIONAL) or single zone (ZONAL).

Reply:
Changed description to above.